Permissions/UITest/Dismiss.m

```diff
@@ -1,4 +1,8 @@
-#import <XCTest/XCTest.h>
+#import "XCTest+CBXAdditions.h"
```

Comment:
Are the changes in this file needed?

Reply:
It was needed to build it. There was build errors due to outdated state of Permissions related sources.